src/radical/pilot/agent/agent_0.py

```diff
@@ -340,77 +351,61 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        self._log.info("Starting the agent services")
+        service_descriptions = self._cfg.services
+        services = list()
+
+        for service_desc in service_descriptions:
+            task = dict()
+            task['origin'] = 'services'
+            task['description'] = TaskDescription(service_desc).as_dict()
+            task['state'] = rps.AGENT_STAGING_INPUT_PENDING
+            task['status'] = 'NEW'
+            task['type'] = 'task'
+            task['uid'] = ru.generate_id('services.%(item_counter)04d',
+                                             ru.ID_CUSTOM, ns="TODO_WHATTOUSE")
+            self._service_task_ids.append(task['uid'])
+            task['pilot_sandbox'] = self._cfg.pilot_sandbox
+            task['task_sandbox'] = self._cfg.pilot_sandbox + task['uid'] + '/'
+            task['task_sandbox_path'] = 'file://localhost/' + self._cfg.pilot_sandbox + task['uid'] + '/'
```

Comment:
@andre-merzky this is wrong. Making path -> `/home/workstation/radical.pilot.sandbox/rp.session.debian-x.workstation.019375.0001/pilot.0000/file:/localhost/home/workstation/radical.pilot.sandbox/rp.session.debian-x.workstation.019375.0001/pilot.0000/services.0021`

I copied this. should this be `self._cfg.pilot_sandbox + task['uid'] + '/'`

Reply:
I may be missing something, but what is wrong?  The line you reference above will not result in a paths like `/home/.../file://...pilot.sandbox`, but rather in a path like `file://localhost/home/.../service.0000/`.  That might be wrong, I don't know, but the point is I don't understand your comment.

Reply:
This is sorted after the last commit I pushed after this comment.